README.md

```diff
@@ -5,8 +5,8 @@
 - Website: https://carvel.dev/kapp
 - Slack: [#carvel in Kubernetes slack](https://slack.kubernetes.io)
 - [Docs](docs/README.md) with topics about diff, apply, gitops, config, _blog posts and talks_ etc.
-- Install: grab prebuilt binaries from the [Releases page](https://github.com/vmware-tanzu/carvel-kapp/releases) or [Homebrew Carvel tap](https://github.com/vmware-tanzu/homebrew-carvel).
-- Backlog: [See what we're up to](https://github.com/orgs/vmware-tanzu/projects/16/views/1?filterQuery=repo%3A%22vmware-tanzu%2Fcarvel-kapp%22).
+- Install: grab prebuilt binaries from the [Releases page](https://github.com/carvel-dev/kapp/releases) or [Homebrew Carvel tap](https://github.com/carvel-dev/homebrew-carvel).
```

Comment:
```suggestion
- Install: grab prebuilt binaries from the [Releases page](https://github.com/carvel-dev/kapp/releases) or [Homebrew Carvel tap](https://github.com/carvel-dev/homebrew).
```

Reply:
Was the correct commit cherry-picked? (I remember making a force push to make this change in the PR for develop)